src/core/qgsquick/qgsquickmapsettings.h

```diff
@@ -160,12 +162,17 @@ class QFIELD_CORE_EXPORT QgsQuickMapSettings : public QObject
     //! Move current map extent to have center point defined by \a layer. Optionally only pan to the layer if \a shouldZoom is false.
     Q_INVOKABLE void setCenterToLayer( QgsMapLayer *layer, bool shouldZoom = true );
 
+    Q_INVOKABLE void setExtentFromPoints( const QVariantList &points );
```

Comment:
sorry for being that guy, but this is the only method in the neighbouring lines without doc.